pkg/cmd/config.go

```diff
@@ -90,6 +96,20 @@ func setConfigValue(key, value string) error {
 	return saveConfig()
 }
 
+func listAllConfig(out io.Writer) error {
+	config, err := unmarshalConfig()
+	if err != nil {
+		return nil
+	}
+
+	// Iterating over structs is hard and there are only two configuration
+	// directives at time of writing. Hard-code the output format for now, and
+	// invest in a more maintainable method in the future.
+	_, err = fmt.Fprintf(out, "user.name=%s\nuser.email=%s\n",
```

Comment:
Viper provides a function for retrieving all the configuration values as a map: `AllSettings`. So instead of hardcoding the configurations keys you could have:
```
	settings := viper.AllSettings()

	for k, v := range settings {
		fmt.Fprintf(out, "%s=%s\n", k, v)
	}
```